components/bybit/actions/linear-market-query-kline/linear-market-query-kline.mjs

```diff
@@ -0,0 +1,57 @@
+import bybit from "../../bybit.app.mjs";
+import { TRIGGER_PRICE_TYPES } from "../../common.mjs";
```

Comment:
Move this to a `constants` file, will be more clear!

```suggestion
import { TRIGGER_PRICE_TYPES } from "../../common/constants.mjs";
```

Reply:
Done